tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -534,6 +534,116 @@ def test_calculated_summary_value_source(mocker, list_item_id):
     )
 
 
+@pytest.mark.parametrize(
+    "list_item_id",
+    [None, "item-1"],
+)
+def test_new_calculated_summary_value_source(mocker, list_item_id):
+    schema = mocker.MagicMock()
+    schema.get_block = Mock(
+        return_value={
+            "id": "number-total",
+            "type": "CalculatedSummary",
+            "calculation": {
+                "operation": {
+                    "+": [
+                        {"source": "answers", "identifier": "number-answer-1"},
+                        {"source": "answers", "identifier": "number-answer-2"},
+                    ]
+                }
+            },
+        },
+    )
+
+    if list_item_id:
+        location = Location(
+            section_id="test-section", block_id="test-block", list_item_id="item-1"
+        )
+    else:
+        location = None
```

Comment:
You should still have a location?. Just the `list_item_id` should be `None`.

Reply:
Updated 👍